api/namex/services/name_request/abstract_name_request.py

```diff
@@ -151,10 +151,10 @@ def generate_nr(cls):
         return nr_num
 
     @classmethod
-    def create_expiry_date(cls, start: datetime, expires_in_days: int, expiry_hour: int = 0, expiry_min: int = 1,
+    def create_expiry_date(cls, start: datetime, expires_in_days: int, expiry_hour: int = 23, expiry_min: int = 59,
                            tz: timezone = timezone('US/Pacific')) -> datetime:
-        """Create an expiry date in given days starting tomorrow."""
-        date = (start.astimezone(tz) + timedelta(days=expires_in_days + 1)) \
+        """Create an expiry date in given days and at 11:59pm."""
+        date = (start.astimezone(tz) + timedelta(days=expires_in_days)) \
```

Comment:
I'm concerned that "start" is in one timezone (eg, Daylight time) but the target date is in another timezone (eg, Standard time). Adding a fixed number of days won't fix the timezone offset.

Or... does the Alchemy client correctly convert and save the date in the db ???

Reply:
outdated comment - closing